var/spack/repos/builtin/packages/fenics/package.py

```diff
@@ -39,7 +40,7 @@ class Fenics(CMakePackage):
         deprecated=True,
     )
 
-    dolfin_versions = ["2019.1.0", "2018.1.0", "2017.2.0", "2016.2.0"]
+    dolfin_versions = ["master", "2019.1.0", "2018.1.0", "2017.2.0", "2016.2.0"]
```

Comment:
Instead of adding this to the list and adding special conditions for master/not-master, why not leave it out of the list and add one extra line for master for each dep? Or add master versions of these deps?

Reply:
Yes, I agree this is better. Fixed in https://github.com/spack/spack/pull/42182/commits/73a7f7c1598e1e440deace0b8ef7e2bb007b2f2f